build.sbt

```diff
@@ -129,6 +136,7 @@ val scalacheck = project.in(file("test") / "scalacheck")
    // Dotty 0.3.0-RC1 crashes when trying to compile this project
   .settings(disableDotty)
   .settings(
+    collectionsScalaVersionSettings,
     fork in Test := true,
     javaOptions in Test += "-Xss1M",
     libraryDependencies ++= Seq(
```

Comment:
Also, a side-effect of the changes in this file is that the project can be imported without errors in IntelliJ.